src/components/ComponentConfiguration.stories.tsx

```diff
@@ -2360,6 +2360,7 @@ export const AddressNL: Story = {
         required: false,
       },
       deriveAddress: false,
+      autoPopulated: false,
```

Comment:
This should be a part of the `defaultValue`, right?
```suggestion
    defaultValue: {
        autoPopulated: false,
    },